pkg/tag/tag.go

```diff
@@ -14,9 +14,11 @@ import (
 // ImageTag is a representation of an image tag with metadata
 // Use NewImageTag to to initialize a new object.
 type ImageTag struct {
-	TagName   string
-	TagDate   *time.Time
-	TagDigest string
+	TagName     string
+	TagNamePart string
```

Comment:
I think this field would need some additional comment on what it actually represents, it is not clear to me.